composer/dag_test_utils/README.md

```diff
@@ -6,7 +6,7 @@ This package is used internally to unit test the validity of all Cloud Composer
 
 Add the following to your `requirements-test.txt` file:
 
-`git+https://github.com/GoogleCloudPlatform/python-docs-samples.git#egg=dag_test_utils&subdirectory=composer/dag_test_utils`
+`cloud_composer_dag_test_utils`
```

Comment:
should google be in the package name or is this agnostic to our stuffs?

Reply:
At the moment, it's agnostic to our stuff, though I could envision it being expanded to be more composer-y in the future. I wasn't sure if there were restrictions on using google in the package name - I'm happy to add it

Reply:
I could go either way on this.  I tend to think being upfront that "this is a google package" is an alright thing, but could also think that by not including google in the name it is more likely to be 'hidden' when folks search for things. :)

Reply:
Okay! I think I'll keep it as is for now for that 'hidden' reason